lib/hades/mentorships/mentor.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Hades.Mentorships.Mentor do
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias Hades.Mentorships.Mentor
+
+  @allowed_skill_areas ~w(Backend Frontend DevOps UX/UI Mobile Fullstack)
+
+  schema "mentors" do
+    field :is_active, :boolean, default: false
+    field :max_mentorships, :integer
+    field :skill_areas, {:array, :string}
+    field :user_id, :id
+
+    timestamps()
+  end
+
+  @doc false
+  def changeset(%Mentor{} = mentor, attrs) do
+    mentor
+    |> cast(attrs, [:is_active, :max_mentorships, :skill_areas])
+    |> validate_required([:is_active, :max_mentorships, :skill_areas])
```

Comment:
Maybe define a variable once the values will be the same. i.e.: `@mentor_attrs ~w(is_active max_mentorships skill_areas)`

Reply:
Good idea!